### PR TITLE
NN-2437 Update court user landing page

### PR DESCRIPTION
**Pull request by jsrobertson**
Updated court user landing page:
image

Moved request booking link:
image

### CODE REVIEW

File: views/courtsVideolink.njk [hunk 1]
```diff
@@ -1,25 +1,30 @@
...
+
+            <div class="govuk-!-margin-bottom-6">
+                <h2 class="govuk-heading-m">
+                    <a href="/prisoner-search" class="govuk-link">Create a video link for a single person</a>
+                </h2>
+                <p class="govuk-body">You'll need the name of who you want to create the video link for and the prison they are in. The service will created a video link booking between a court and a prison.</p>
+                <p class="govuk-body">If the person is not currently in a prison, you'll need their name, date of birth and the prison they are being sent to. A request will be sent to the prison to complete the booking.</p>
+            </div>
+
+            {# <div class="govuk-!-margin-bottom-6">
```
Comment (jsrobertson) [inline views/courtsVideolink.njk]: Put this in but commented out as page/journey doesn't exist yet.

File: backend/routes.js [hunk 2]
```diff
@@ -213,8 +213,6 @@ const setup = ({ elite2Api, whereaboutsApi, oauthApi, communityApi }) => {
   router.get('/videolink', async (req, res) => {
     res.render('courtsVideolink.njk', {
       user: { displayName: req.session.userDetails.name },
-      homeUrl: '/videolink',
-      title: 'Video link appointment booking',
     })
```
Comment (Nimphal) [inline backend/routes.js]: Ooh, the homeUrl is used in the header for where clicking the logo will take them to as well as the breadcrumbs. Have we decided we want them to go to DPS if they click the logo?

File: backend/tests/requestBooking.test.js [hunk 3]
```diff
@@ -40,7 +40,6 @@ describe('Request a booking', () => {
         expect(res.render).toHaveBeenCalledWith('requestBooking/requestBooking.njk', {
           errors: undefined,
           formValues: undefined,
-          homeUrl: '/videolink',
```
Comment (Nimphal) [inline backend/tests/requestBooking.test.js]: Same here.

File: backend/routes.js (hunk 2 above)
Comment (jsrobertson) [reply backend/routes.js]: Ahh, my bad.  Thought it was just used in the breadcrumbs. Will change.  Cheers

**Pull request merged**